agr_java_core/src/main/java/org/alliancegenome/neo4j/entity/DiseaseAnnotation.java

```diff
@@ -76,7 +76,6 @@ public class DiseaseAnnotation extends ConditionAnnotation implements Comparable
 	@JsonView({View.DiseaseAnnotation.class})
 	private List<Gene> orthologyGenes;
 
-	@JsonIgnore
```

Comment:
@cmpich yes... I did make this change. There is issues with turning this on.

Reply:
Can we try running this? The approach in not serializing PublicationJoin is not going to work as I need it for the Download logic.